mesos-simulation/src/test/scala/mesosphere/mesos/scale/MetricsResult.scala

```diff
@@ -0,0 +1,160 @@
+package mesosphere.mesos.scale
+
+import play.api.libs.json._
+import org.apache.commons.io.FileUtils
+import java.io.File
+
+sealed trait Metric {
```

Comment:
I would put the support objects either in MetricsResult or better in a separate file (for reuse).